modules/about-centralized-auth-oidc.adoc

```diff
@@ -0,0 +1,22 @@
+:_module-type: CONCEPT
+
+[id='about-centralized-auth-oidc_{context}']
+= About centralized authentication Gateway API
+
+A Gateway API with centralized authentication centralizes ingress for all services behind a single domain, providing the following advanced capabilities:
+
+* Centralized authentication: A single authentication service requiring only one client ID and secret from the external OIDC Identity Provider (IDP). 
+* Simplified backend services: Backend services assume all incoming traffic is authenticated and contains necessary user headers. 
+* Authorization handling: Services still handle authorization at the service or pod level using sidecars like `kube-rbac-proxy`.
+* Encrypted Communication: Traffic from the gateway to the backend services is fully encrypted with Transport Layer Security (TLS). 
+
+The Gateway API is implemented via an Istio Gateway on {openshift-platform} (OCP) 4.19 and later. Since the Istio Gateway is built on the Envoy Proxy, it provides access to powerful Envoy-specific Custom Resource Definitions (CRDs), such as `EnvoyFilter`. The `opendatahub-operator` manages the deployment of `kube-auth-proxy`. The Operator then configures the Istio Gateway to use this service via an `EnvoyFilter` Custom Resource (CR).
+
+For more information on supported OpenID Connect (OIDC) identity providers, see OCP documentation on link:https://docs.redhat.com/en/documentation/openshift_container_platform/{ocp-latest-version}/html/authentication_and_authorization/external-auth#external-auth-providers_external-auth[Direct authentication identity providers]
```

Comment:
According to the RHSSG, we shouldn't ever use OCP as an abbreviation on its own. It should always just be {openshift-platform}, even if you parentheses the platform as you do earlier: https://redhat-documentation.github.io/supplementary-style-guide/#red-hat-openshift-container-platform